exercise/submission_models.py

```diff
@@ -222,7 +222,10 @@ def set_ready(self):
         # Fire set hooks.
         for hook in self.exercise.course_module.course_instance \
                 .course_hooks.filter(hook_type="post-grading"):
-            hook.trigger({ "submission_id": self.id })
+            hook.trigger({
+                "submission_id": self.id,
+                "exercise_id": self.exercise.id,
```

Comment:
should also include course_id (course_instance.id). and url to plus.cs.hut.fi (well, to installation), probably in field `site`.